### ISSUE TITLE
Switching language resets techs

### ISSUE THREAD

**Issue by GoogleCodeExporter** [sunfall/endgame-singularity#99]
```
What steps will reproduce the problem?
1. Finish game
2. Switch language and look at research

What is the expected output? What do you see instead?
Expected that the researches left to study would only be translated, found 
them reset too.

What version of the product are you using? On what operating system?
Windows, latest version

Please provide any additional information below.
I also changed back to en_US to see if it corrects, and it did not.


```

Original issue reported on code.google.com by `Avaera8820` on 14 Oct 2009 at 3:03

**Comment by GoogleCodeExporter** [comment]
```
Here goes a patch that fixes this
```

Original comment by `rodrigo....@gmail.com` on 5 Mar 2012 at 1:47

Attachments:
* [0001-fix-techs-being-reset-if-language-is-changed-mid-game-(Closes-#99).patch](https://storage.googleapis.com/google-code-attachments/endgame-singularity/issue-99/comment-1/0001-fix-techs-being-reset-if-language-is-changed-mid-game-(Closes-#99).patch)

**Comment by GoogleCodeExporter** [comment]
```
Fixed in git commit 3c59ba01..
```

Original comment by `phil.bor...@gmail.com` on 29 Mar 2012 at 6:48
* Changed state: **Fixed**
